easybuild/easyblocks/o/openfoam.py

```diff
@@ -447,7 +447,17 @@ def sanity_check_step(self):
             'dirs': dirs,
         }
 
-        super(EB_OpenFOAM, self).sanity_check_step(custom_paths=custom_paths)
+        # test the OpenFOAM environment in BASH
+        load_openfoam_env = "source $FOAM_BASH"
+        # test monitorFoam (exit code will only be 0 if all dependencies are met)
+        test_monitorFoam = " && ".join([load_openfoam_env, "foamMonitor -help"])
```

Comment:
@lexming `OpenFOAM-Extend` doesn't provide `foamMonitor`, so this needs to be done conditionally

Done in https://github.com/lexming/easybuild-easyblocks/pull/3